packages/material-ui/src/Select/SelectInput.js

```diff
@@ -495,6 +495,14 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         anchorEl={displayNode}
         open={open}
         onClose={handleClose}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'center',
```

Comment:
Why "center"? It makes our `<Select />` looks like this:

![image](https://user-images.githubusercontent.com/5390719/130719260-7768a978-352f-45ec-846c-1a57df146d16.png)